internal/metrics/metrics.go

```diff
@@ -86,5 +103,10 @@ func RegisterStatuserMetrics() {
 }
 
 func RegisterApiMetrics() {
+	// no metrics
+}
+
+func RegisterWorkerMetrics() {
 	prometheus.MustRegister(JobQueueSize)
+	prometheus.MustRegister(BackgroundJobDuration)
```

Comment:
Just for consistency
```suggestion
	prometheus.MustRegister(JobQueueSize, BackgroundJobDuration)
```

Reply:
@adiabramovitch could you do a follow-up for it? If it's not a blocker, I'd like to get this in, so we start capturing the metrics and Lukas will not get to it of a week now :)

Reply:
Addressed in #443